src/dask_awkward/layers/layers.py

```diff
@@ -117,6 +117,19 @@ def io_func_implements_report(func: ImplementsIOFunction) -> bool:
     return hasattr(func, "return_report")
 
 
+class AwkwardTokenizable:
+
+    def __init__(self, ret_func, parent_name):
+        self.parent_name = parent_name
+        self.ret_func = ret_func
+
+    def __dask_tokenize__(self):
+        return (AwkwardTokenizable, self.parent_name)
+
+    def __call__(self, *_, **__):
+        return self.ret_func
```

Comment:
```suggestion
        return self.ret_func()
```
?

Reply:
the original code says otherwise `io_func=lambda *_, **__: new_return,`

Reply:
right that is a value not a function :D